app/views/podcast_metrics/_downloads_card.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= turbo_frame_tag "downloads" do %>
+  <% if episode_rollups.present? %>
+    <div class="card shadow border-0"
+      data-controller="apex"
+      data-apex-id-value="<%= SecureRandom.uuid %>"
+      data-apex-type-value="line"
+      data-apex-trunc-value="<%= date_trunc %>"
+      data-apex-series-value="<%= parse_episode_downloads(episode_rollups, date_start, date_end, date_trunc).to_json %>">
+      <div class="card-header card-header-light d-flex justify-content-between">
+        <h2 class="card-title h5 flex-grow-1">Downloads</h2>
+        <%= render "dateview", form: form, episode_rollups: episode_rollups, date_start: date_start, date_end: date_end, date_trunc: date_trunc %>
+      </div>
+      <div class="card-body">
+        <div class="row d-flex flex-wrap gx-4">
+          <div class="col">
+            <div data-apex-target="chart"></div>
+          </div>
+          <div class="col align-items-center">
+            <div class="col my-4">
+              <table class="table table-striped shadow rounded">
+                <thead>
+                  <tr class="table-secondary">
+                    <th>Episode</th>
+                    <th>Publish Date</th>
+                    <th>Downloads</th>
+                    <th>All-time</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="fw-bold">All Episodes</td>
```

Comment:
Eventually will want to i18n all this.  But I'm fine with a separate ticket for that.